llvm/include/llvm-c/Core.h

```diff
@@ -3434,6 +3441,13 @@ void LLVMMoveBasicBlockBefore(LLVMBasicBlockRef BB, LLVMBasicBlockRef MovePos);
  */
 void LLVMMoveBasicBlockAfter(LLVMBasicBlockRef BB, LLVMBasicBlockRef MovePos);
 
+/**
+ * Obtain the context to which this basic block is associated.
+ *
+ * @see llvm::Module::getContext()
+ */
+LLVMContextRef LLVMGetBasicBlockContext(LLVMBasicBlockRef BB);
```

Comment:
This shouldn't be needed. A BasicBlock is also a Value.

Reply:
I am planning to use this via [llvm-sys.rs](https://gitlab.com/taricorp/llvm-sys.rs), and that would make the API more consistent both in C and in the rust bindings.

Reply:
Just to be clear, my point here is that you could write `LLVMValueGetContext(LLVMBasicBlockAsValue(BB))` to achieve the same -- do we really need a shorthand for it?

Reply:
Was not aware initially of this, but yeah it makes sense to not include the additional function. I have removed it in the latest version.